Munitions Express

scrubmcnoob opened the pull request:
Shotgun ammunitions actually available without having to hack a machine as security.

Just like true American fashion, like good ol' walmart, all shotgun purchases are in bulk buy and is more expensive point to metal ratio than just printing it out of a hacked autolathe. So there is still an incentive to hack if you wish to save every little point and don't need 42 rounds.

Dramatically ups the price of the combat shotgun as it's really powerful and for whatever reason three automatic ballistic guns with bandoliers were cheaper than two eguns by five points.
Adds in riotshotguns as a cheaper solution.
Adds in the dragonsbreath ammo box.
Shotgun ammo_boxes are now named speedloaders and resprited to reflect this. Sprites clearly display how many shells are in each speedloader
http://i.imgur.com/ppKQhCW.png
http://i.imgur.com/ZSW4bkF.png

If you're wondering where the hell I got an idea for these things, from this.
https://www.youtube.com/watch?v=USdFTFG0iCI
Obviously had to make them not look like belt items in the video and had to make them hold seven shells. Made the holder itself look noticeable as I don't want an empty holder to look like shite.
Yee-Haw

:cl:
tweak: Combat Shotguns now more expensive in cargo.
tweak: Shotgun ammo_boxes are now named and sprited as speedloaders that can speedload into a shotgun. Normal shotgun ammo boxes are just storage items for shells.
rscadd: Adds in shotgun ammos readily available to be ordered from cargo in bulk and allows the ordering of Riot shotguns.
rscadd: Adds in a Dragonsbreathe shotgun ammo box.
fix: Tranq darts now have a sprite.
/:cl:

Anasari commented:
I support this PR as it make t easier to organize ammo for shotguns. 

It also fixes the imbalance problem with combat shotgun being ridiculously cheap

Jovaniph commented:
Why is wood being removed?

Lady-Luck commented:
Can the egun crate be upped to 3 eguns?

scrubmcnoob commented:
Ignore the wood thing, minor mistake, and we could do 3 eguns? I mostly want to see how people like the initial PR before anything else.

Lady-Luck commented:
just getting 2 eguns in a crate feels really... meh?

scrubmcnoob commented:
Three laser guns to the crate.

Anasari commented:
MOAR DAKKA

Fox-McCloud commented on code/datums/supplypacks.dm:
```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
+	name = "Riot Shotguns Crate"
+	contains = list(/obj/item/weapon/gun/projectile/shotgun/riot,
+					/obj/item/weapon/gun/projectile/shotgun/riot,
+					/obj/item/weapon/gun/projectile/shotgun/riot,
```
I'm a bit skeptical of this, to be honest; the main drawback of shotguns is that not every single officer can carry one around without consequene--only two officers.

Making them commonly available not only removes that drawback, but the pricepoint of these is going to encourage players to go with this over the combat.

shazbot194 replied:
I'm not disagreeing shotguns should stay somewhat rare, but I have to say these are probably going to be as rare as combat shotguns are now if not more from the cost. If you really think about it, there is almost no reason all officers don't carry around an auto-rifle or have a combat shotgun other then SoP and the fact no-one buys them until they are needed.

Fox-McCloud commented on code/datums/supplypacks.dm:
```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	contains = list(/obj/item/weapon/gun/energy/gun,
-					/obj/item/weapon/gun/energy/gun)			// Only two guns to keep costs down
+					/obj/item/weapon/gun/energy/gun,
+					/obj/item/weapon/gun/energy/gun)
```
But why?

Fox-McCloud commented:
I'm not to keen on this to be honest; it's encouraging the proliferation of ballistics---and on a very cheap scale.

Thematically, the station doesn't use ballistics much, and the ones that it has available to it come at a downside (the sec rifle has a poor firing rate and the ammo is hard to come by---combat shotguns are expensive and can't be put in backpacks, and their ammo is even more expensive).

This makes riot shotguns more common, which are really just poor man's revovlers with slugs...and with how dirt cheap the ammo is, officers have no reason to not run around with boxes upon boxes of ammo.

This is also going to really screw with the crew vs Nuke Ops dynamic.

FlattestGuitar commented on code/datums/supplypacks.dm:
```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
+					/obj/item/weapon/storage/belt/bandolier,
+					/obj/item/weapon/storage/belt/bandolier,
+					/obj/item/weapon/storage/belt/bandolier)
+	cost = 30
```
I really feel like this should be higher. That's just six sheets of plasma for three death machines.

Same goes for all other weapons.

FlattestGuitar commented on code/datums/supplypacks.dm:
```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
+	cost = 30
+	containername = "riot shotgun crate"
+
+/datum/supply_packs/security/armory/ballistic
```
Same path as the previous one. One of these won't show up in game. Chaaaange.

FlattestGuitar commented on code/datums/supplypacks.dm:
```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 					/obj/item/weapon/storage/belt/bandolier,
 					/obj/item/weapon/storage/belt/bandolier)
-	cost = 20
+	cost = 40
```
I really feel like this should be higher. That's just six sheets of plasma for three death machines.

Same goes for all other weapons.

Ssirius commented:
Combat shotguns(and shotguns in general) are way too common for how strong they are as it is, they have become a go to weapon for anything humanoid antag related, all it takes most of the time is a single shot to effectively kill anyone that cant get broken head or chest bones mended.
I'm also not too keen on adding cheaper riot shotguns to cargo either for the same reason.

With the combat shotgun cost nerfs and the autolathe cost dynamic with the ammo crate I support this, but would much rather see shotguns become a lot less viable in general.

sothangel commented:
Fox frankly doesn't know the reality of the situation if he's saying that combat shotguns are expensive enough already and not used that often. Combat shotguns are THE go to weapon to order for War Ops, Xenomorphs, Wizards. Not being able to have a bag is completely irrelevant if you can kill someone all the antags in a few shots. I'm completely for the increase in combat shotgun price (Would probably even triple it if riot shotguns are being placed in the menu as well) 

I don't feel dragonsbreath is appropriate for the crew to have, and I'm not sure about the price/content ratio of the ammo crates, would need to see some numbers regarding how quickly they can be amassed after certain times, etc etc.

Lady-Luck commented:
Nah. I soloed a xenomorph queen with a combat shotgun. They're too cheap for how good they are.

scrubmcnoob commented:
> I'm not to keen on this to be honest; it's encouraging the proliferation of ballistics---and on a very cheap scale.

The current price for three combat shotguns is only 20 points. A crate for two laser guns is 25 points. And you saying this will make ballistics go up on a very cheap scale? This is doubling the price of combat shotguns and adding in riot shotguns which are still more expensive than current combat shotguns.
Also, a combat shotgun can drop a nukie/anyone with a quick spam click Even with their really cheap price, the station is not always littered with them and it's not a huge problem.
Saying this PR would make the problem a ton worst is something that is really unfounded.

> This is also going to really screw with the crew vs Nuke Ops dynamic.

I doubt that will be the case at all given combat shotguns are only 20 points and you still see nukies winning plenty. People do order these guns yes but the PR will make it even harder to acquire these guns.

> I'm also not too keen on adding cheaper riot shotguns to cargo either for the same reason.

The 'cheaper' riot shotguns are cheaper to combat shotguns in this PR. They're actually more expensive than current combat shotguns which are only 20 points and again, it's not a major problem. 


If you want, I can easily raise the prices of the shotguns even higher. And the munitions are there because currently the only way for security to get more shells in their armory is by using a hacked auto lathe which is just ridiculous. Even with this PR though, it's still much cheaper to just order metal from cargo, insert it into a hacked auto lathe, and print the shells out.

And I'm going to reiterate that a combat shotgun crate, as of right now, is only 20 points and the station isn't falling apart from them whereas this PR will raise the price to 40 points making it even harder.

SamHPurp commented on code/datums/supplypacks.dm:
```diff
@@ -1193,6 +1193,13 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	cost = 10
 	containername = "glass sheets crate"
 
+/datum/supply_packs/materials/wood30
```
Trying to revert my PR, you jerk.

SamHPurp commented:
Hells frozen over and I agree with Fox.

This doesn't seem like a good idea, and really really screws with balance in already teetering Gamemodes such as War Ops.

Lady-Luck commented:
But War Ops could really use the help? So making combat shotguns not so cheap would be good for balance.

scrubmcnoob commented:
How does this screw the balance with challenge ops?
Am I living in an alternate dimension where higher costing things are good and actually help the crew?
I'm literally making it harder for crew to buy combat shotguns. The price is doubled.
Unless you want crew to buy combat shotguns for five points?

Anasari commented:
From my own experience as a warden / HoS, most security officers I've seen prefer the overwhelming firepower of the combat shotgun rather than the compactness of a riot shotgun. Riot shotgun don't have the ability to burst down anyone in its way, even if it can be carried around in backpack. I think this is worth giving a try.

If it turns out that being able to carry a shotgun in your backpack is worth more than rapid firepower, then the cost of riot shotgun can always be adjusted upward

Spacemanspark commented:
Incoming rant from Flattest in 2 minutes...

FlattestGuitar commented:
:+1:

Spacemanspark commented:
Also hi everybody I hope you like e-mails

Doukhan commented:
Buckshot shotguns are the shit, they murderfuck pretty much anything in a matter of seconds and it's not uncommon to have little squads of sec/command roll up to and deal with "problems" when red alert gets called, it's quite a bit of a buzzkill when the antags all get lol-shotted.

I like the idea behind this PR but all it's going to do is make the presence of I-win-guns ubiquitous, they're already the go-to option and bulk ammo will only make them the objectively superior choice to spam order whenever there's anything that needs pew-pewing.

Lady-Luck commented:
>I like the idea behind this PR but all it's going to do is make the presence of I-win-guns ubiquitous, they're already the go-to option and bulk ammo will only make them the objectively superior choice to spam order whenever there's anything that needs pew-pewing.

Uh, they can already get as many buckshot shells as they need from a hacked autolathe. it's cheap and easy to do. This just adds a 'legitimate' method of acquiring the shells.

Fox-McCloud commented:
>Fox frankly doesn't know the reality of the situation if he's saying that combat shotguns are expensive enough already and not used that often.

It's not the price increase I take issue with; as a matter of fact, I didn't even comment on the price increase being a negative thing for the Combat Shotguns (because they *are* pretty ridiculous, as @Ssirius noted); I take issue with adding more ballistics to sec and giving them an even easier and (I believe) cheaper way of acquiring ammo.

Lady-Luck commented:
I think I'd still prefer a hacked lathe when it comes to ammo because it's instant and doesn't require anyone to open the box.

scrubmcnoob commented:
> Buckshot shotguns are the shit, they murderfuck pretty much anything in a matter of seconds and it's not uncommon to have little squads of sec/command roll up to and deal with "problems" when red alert gets called, it's quite a bit of a buzzkill when the antags all get lol-shotted.

This PR makes the automatic combat shotguns, that start loaded with buckshot, more expensive.
This makes those automatic 'lol-shotguns' double the price.

> I like the idea behind this PR but all it's going to do is make the presence of I-win-guns ubiquitous, they're already the go-to option and bulk ammo will only make them the objectively superior choice to spam order whenever there's anything that needs pew-pewing.

Again, this makes combat shotguns more expensive. Also, you can spam more shells from the hacked autolathe overtime quite easily and it's much cheaper point wise than that of ordering boxes. This adds in a legit way of obtaining shotguns shells without having to hack a machine.

> @scrubmcnoob The price is kinda irrelevant since cargo are always swimming in points (usually floating around 1000 by the half way mark), buckshot absolutely wrecks nukies, these changes make it waaay easier for an enterprising crew to arm all of sec with minimal effort.

This is if mining does it's job and science doesn't hog all the resources to themselves as soon as they can, which they usually do. If, however, points are seemingly irrelevant to cargo, than perhaps it should be looked at and the price for plasma sheets re-adjusted to fix this? I find many times that cargo may be lacking point wise.

> One or two officers who know what they're doing with one can stomp a full squad of ops assuming the syndies aren't tacticool masters and don't get a lucky shot off, imagine the whole department rushing at them armed to the teeth and you'll see why it'll screw with muh balance.

This can already easily happen with the under priced combat shotguns but nukies still manage to win from time to time.

> I take issue with adding more ballistics to sec and giving them an even easier and (I believe) cheaper way of acquiring ammo.

This doesn't really add more ballistics because sec already has riot shotguns in the armory and already has access to order under priced combat shotguns, we're just allowing them to order another type which has much slow fire rate but is more compact. If you want, I can raise the riot shotguns to match the combat shotguns or be five points below the combat shotguns. I can also raise the overall prices of both shotguns if you so desire.
And the ammo being 'cheaper' to acquire is objectively false as it's cheaper to print them out of an auto lathe. I think it's around .3ish points per shell out of a lathe, and around .9ish points per shell with ordered ammo. And whether it's easier or not is dependent on the situation. If you got an engineer to hack the lathe, it's better to load up the processing queue to full, do some things, spam it back to full and collect already made shells. 
Again, this just adds in a more legit way of actually acquiring ammo in security's perspective without having to grab an engineer to hack the lathe.
Also hacked lathes allow literally anyone to grab ammo but with these boxes, a person with armory access needs to open them. This makes it even harder for crew to obtain shotgun ammo as there won't be a hacked autolathe lying around as much when security no longer needs to rely on them.

Fox-McCloud commented:
I'm not concerned about the crew in this instance; I'm worried about security---what you mention (a legit way to acquire it) is inherently making it easier for them to access; they don't have to find that engineer, they don't have to worry about the queue; they just order it and they get 6 boxes of ammo.

Yeah, the price thing I was wrong on; I checked it out; the crate has roughly 84 sheets of metal worth of supplies for 35 points (which comes out to be 17 points worth of metal). That said, I still think this is making it too easy to acquire, not to mention that, again, ammo boxes are, in my opinion, absolutely horrible because they trivialize reloading with a single click.

The slow rate of fire is virtually negligible in the hands of a skilled player (or who uses some form of `attack_self` keybinding) and for that small price he gets a gun that can fit in his backpack.


Sec already has access to ballistics, and it causes problems (particularly with nuke ops and wizards); I don't think they need access to *more*.

scrubmcnoob commented:
> I'm not concerned about the crew in this instance; I'm worried about security---what you mention (a legit way to acquire it) is inherently making it easier for them to access; they don't have to find that engineer, they don't have to worry about the queue; they just order it and they get 6 boxes of ammo.

Why the hell should security have to hound an engineer/look for gloves to hack a lathe to get more ammo in the first place? Why can't they just order it?

> Yeah, the price thing I was wrong on; I checked it out; the crate has roughly 84 sheets of metal worth of supplies for 35 points (which comes out to be 17 points worth of metal). That said, I still think this is making it too easy to acquire, not to mention that, again, ammo boxes are, in my opinion, absolutely horrible because they trivialize reloading with a single click.

Mind showing me your math for this so I may adjust prices accordingly? After reading your post, I realise that I forgot to account for the points you get back if you send the crate back.
Also with the opening containers inside your backpack PR, you can just as easily print bandoliers from botany for yourself and fill a bag with them. And I believe bandoliers hold more shells than an ammo box.

> The slow rate of fire is virtually negligible in the hands of a skilled player (or who uses some form of attack_self keybinding) and for that small price he gets a gun that can fit in his backpack.

This is assuming every officer who picks up a shotgun, or anyone for that matter, is a god player and will know how to use the gun at full efficiency but we know this is not true. The gun is slower than the combat as you just click as fast as you can with the combat. 

> Sec already has access to ballistics, and it causes problems (particularly with nuke ops and wizards); I don't think they need access to more.

Sec can already order a load of combat shotguns for dirt cheap and those antags still do fine.
Again, give me price suggestions that you may have, and I will incorporate them. There is no reason that security has to hack a machine just to get extra ammo when they should be able to order some.

scrubmcnoob commented:
Alright, I'm going place my math here for ease.
1 metal sheet=2000 mass in autolathe
1 shotgun shell=4000 mass=2 metal sheets
10 cargo points=50 sheets of metal=25 shells

If you order an ammo crate, you get 42 shells for 45 points.
If you used a hacked autolathe, you could've gotten over a hundred.

So I increased the prices even more on all ammos and shotguns. The riot shotgun is now the same price as the combat shotgun seeing it was argued that it's compactness equals to the rate of fire of the combat shotgun.

This makes it significantly pricier than current code to order shotguns from cargo.
I have no way of comprehending how this makes it easier to obtain shotguns unless I learned math wrong or did the math wrong.

Lady-Luck commented:
I think Fox is mostly taking issue with the ammo boxes which are by far the most convenient method of reloading.

scrubmcnoob commented:
Bandoliers do the same job and can easily be printed from botany.

Lady-Luck commented:
Do bandoliers do instant full refill though? And can they be stored in bags? I'm honestly not sure.

scrubmcnoob commented:
Bandoliers can be stored in bags and accessed from the bags itself, meaning you never need to take it out and place it into your hand.

Lady-Luck commented:
But can you click a bandolier onto a shotgun to instantly reload it?

Fox-McCloud commented:
>Bandoliers do the same job and can easily be printed from botany.

Not they don't. It's not the holding of lots of ammo I take issue with, it's the two clicks (click container, click gun), to go from 0 ammo clear back up to full (minus chambered), I take issue with.

Ballistics already "recharge" faster than energy guns; ammo boxes push that to the point of ridiculousness.

(and if you're going for mai realisms; you couldn't even remotely reload that quickly using a box; a revolver with a speedloader? sure).

scrubmcnoob commented:
@Fox-McCloud if shotgun boxes are the main issue, would you be against me changing shotgun ammo boxes, so they don't mimic a speedloader, and make it where you open it like a bandolier and take individual shells out? So it's like the bandolier except it only holds seven rounds.

jughu commented:
>adds buckshots = E z kill

>adds slug ???

>adds dragonbreath i guess it can be incendiary...

>DOESNT EVEN ADD RIOT SHOTGUN AMMO

Fox-McCloud commented on code/datums/supplypacks.dm:
```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 					/obj/item/weapon/storage/belt/bandolier,
 					/obj/item/weapon/storage/belt/bandolier)
-	cost = 20
+	cost = 45
```
IMO, this is still too cheap. I'd bump it up to 80.

Fox-McCloud commented:
IMO, I believe all shotgun ammo boxes should be standard boxes with 7 rounds in them.

I'm still not sold on the inclusion of lethal rounds to cargo, mostly because I don't want to encourage the proliferation of ballistics to the crew, because of how strong they are; aside from the lack of a recharge mechanic, they're objectively better weapons.

scrubmcnoob commented:
Alright, that's something I actually want in too @Fox-McCloud Shotgun boxes shouldn't be magical speedloaders but rather just boxes you need to pull shells out of. That's a con to using shotguns, you need to load individual shells unlike every other ballistic which has a speed loader/magazine/clip. I'll get to making it happen.

@jughu no point in adding beanbag rounds as you can already mass produce those rounds from an unhacked autolathe for very cheap

And again, as I showed with my math, it's much much cheaper to just print rounds off a hacked autolathe but I want to give security a different option to get shotgun ammo besides just using a hacked autolathe.

Fox-McCloud commented:
I think ultimately, perhaps, the point you may be missing, that I"m attempting to make here is...

I think lethal shotgun round should be difficult to get ahold of and tedious--something that security should generally be persuaded to utilize, and if they do make the choice, it should be inconvenient for them and time consuming.

Reason being is, I think it's healthier for the crew to be reliant on and be persuaded to go with laser guns as opposed to conventional firearms---likewise, I think it best that the ones they can access have downsides to them (ie: slow firing auto-rifles, the bulky combat shotguns, or the extremely limited supply riot shotguns).

scrubmcnoob commented:
I'm pretty sure the massive fucking price rip off of the ammo is already persuasion enough. Security will now have to argue with disgruntled QMs who don't want to lose their points. 
And why does there need to be this notion to make getting ammo just an annoyance? It honestly  just depends on the engineers/people with gloves on that round. So you may deal with hell getting someone to hack the autolathe or find someone to do it within a minute or find that's it already is hacked. And if the autolathe is hacked, printing shells is super easy.

This PR makes it a fixed probability of the point cost, no longer dependent if someone with gloves is nearby.

Lady-Luck commented:
I was in support of this PR initially, but clearly things are getting way too nerf-happy.

scrubmcnoob commented:
Hardly nerf happy.
They make shotguns more sensible as they currently trump all other weapons you can order in the cargo equipment list.
There is no reason, as of right now, not to order combat shotguns for weapons.
/o7

Lady-Luck commented:
Putting multiple different nerfs in place and merging them at the same time is a recipe for disaster and it's exactly what fucking destroyed genetics. Everyone agreed it should be nerfed, but rather than take a measured approach, two different systems to reduce the power were put into place simultaneously.

We're now looking at losing ammo boxes, substantially lower buckshot damage, potentially 80 point combat shotgun crates, and perhaps losing lathe printing of slugs, all going on at the same time.

scrubmcnoob commented:
Stop assuming.
Hacked lathe printing is staying.
Combat shotguns need a major price buff as you can spam click anything to death with them in seconds.
Ammo boxes are staying, but they will no longer be magical speedloaders but rather just item holders. You can still use them from your bag like bandoliers. Shotguns need some type of downside to make them less lucrative than everything else, taking out their prespawned speed loaders is something they can lose.

Buckshot damage reduction is not apart of this PR.

Lady-Luck commented:
>Buckshot damage reduction is not apart of this PR.

No, it's part of a related PR, this PR is even mentioned as a basis in the description.

Fox has already stated he does not like the lathe printing and wants to get rid of the ammo boxes in the vault, which would make it insanely tedious to unload all the basic rubber shells from the starter riot shotguns and then make them good for distribution.

There is just too much going on at the same time here. Slow the hell down.

scrubmcnoob commented:
Or you simply just insert the shells. I hardly, if ever, use those shotguns boxes, and mainly have bandoliers in my bag. I can load shotguns just fine. And there are still going be boxes, just an actual storage item now instead of a magazine.
And hardly much going on, only a few tweaks to shotgun stuff and additions to cargo.

Lady-Luck commented:
>Or you simply just insert the shells. I hardly, if ever, use those shotguns boxes, and mainly have bandoliers in my bag. I can load shotguns just fine. And there are still going be boxes, just an actual storage item now instead of a magazine.

Are you speaking from the perspective of a warden here having to distribute weapons at red alert? It's already quite tedious and slow, adding about thirty clicks with tiny items isn't exactly going to help that.

My point is too many different nerfs are happening at the same time and I don't like it. Everyone can agree shotguns are too good right now but we certainly have a tendency to nerf shit into the ground.

scrubmcnoob commented:
Yes, I main security as HOS often.
Has the thought occurred to you that the warden can simply place out the shotguns with ammo boxes on the same tile and allow the officers to load the shotgun themselves? I believe they are capable enough of such a task if the warden doesn't have enough time tot his.

And what is nerfing shotguns to the ground?
The ammo box nerf? 
There's only three ammo boxes, for buckshot, that spawn and those are in the armory.
Upping the cargo points?
The shotguns are still going be powerful individually, just not as many can be bought.
Buckshot being nerfed?
They're still 70 damage which means two shots can still crit a person.

Lady-Luck commented:
No nerf individually sounds unreasonable. It's them all bundled together that has me worried.

scrubmcnoob commented:
Why do they worry you though.
I just broke them all down for you.
You're making them out to be much more of a problem than they are.

Lady-Luck commented:
Is there something confusing about my point of view? Shotguns do need to be taken down in power, but we have a tendency to overdo it.

Coming at it from so many different angles all at once before we see how even one change plays out is something that has happened many times before and it can lead to undesirable results.

scrubmcnoob commented:
Okay, what are we 'over-doing' this time?
Making shotgun boxes no longer magazines? There are only three on the map and you have no other way to get them.
Upping prices?
What's is too much nerf here? Break it down for me, tell me which part is being overdone and how we can meet an equilibrium for balance.

Lady-Luck commented:
Too many different types of nerfs going on at the same time is the problem.

scrubmcnoob commented:
>removes three shotgun ammo boxes from autoloading shotguns
>makes shotguns more expensive
>buckshot nerfed to 70 damage

>adds riot shotguns that can be ordered
>adds ammo crates that can be ordered

I see this rather balancing and fine. They are still going be gat 9000s.

scrubmcnoob commented:
OP updated to reflect new changes.

Fox-McCloud commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -152,6 +158,132 @@
+/obj/item/weapon/storage/box/buck
+	name = "Ammunition Box (Buckshot)"
+	desc = "A small box capable of holding seven shotgun shells."
+	materials = list(MAT_METAL=1750)
```
This shouldn't have materials; it's a box.

Fox-McCloud commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -152,6 +158,132 @@
+	desc = "A small box capable of holding seven shotgun shells."
+	materials = list(MAT_METAL=1750)
+	icon_state = "buckshotbox"
+	origin_tech = "combat=2"
```
nor should it have an origin tech...it's a box...with shells in it.

Fox-McCloud commented on code/modules/projectiles/ammunition/boxes.dm:
```diff
@@ -63,48 +63,6 @@
 	ammo_type = /obj/item/ammo_casing/n762
 	max_ammo = 14
 
-/obj/item/ammo_box/shotgun
```
Don't remove these.

scrubmcnoob commented:
Alright.
Re-added shotgun ammo_boxes as asked, and placed them back into the armory, but now they are named and sprited as shotgun tube loaders. The cargo containers will only give you normal shotgun ammo boxes, not tube loaders.

Why did I do this? Well it's really silly and inconsistent if two boxes that are identical do two completely different things, one being a normal storage container and another being a speed loader. Corrected this issue with making them tube loaders. 

Based them off this old guy's invention
[https://www.youtube.com/watch?v=M3Hs4do69ms](url)
yee-haw
picture in OP

Fox-McCloud commented:
those sprites don't look so good. I'd just use the box sprites that we used to use; players will adjust, soon enough, especially considering that the ammo boxes are pretty much unobtainable now, anyway.

Fox-McCloud commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -137,10 +137,14 @@
+	name = "Ammunition Box (Slug)"
+	desc = "A small box capable of holding seven shotgun shells."
+	icon_state = "slugbox"
+	storage_slots = 7
```
redundant.

Fox-McCloud commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -137,10 +137,14 @@
+	desc = "A small box capable of holding seven shotgun shells."
+	icon_state = "slugbox"
+	storage_slots = 7
+	can_hold = list(
```
Don't bother; a box with its own sprite is more than enough.

Fox-McCloud commented on code/modules/projectiles/ammunition/boxes.dm:
```diff
@@ -63,43 +63,55 @@
-/obj/item/ammo_box/shotgun
-	name = "Ammunition Box (slug)"
-	icon_state = "slugbox"
+/obj/item/ammo_box/ashotgun
```
Why the change to pathing?

Lady-Luck commented:
Really liking the way the PR is now. I think the tube loaders are a great idea, but could use better sprites. Don't really understand the opposition to riot shotguns, they are significantly less powerful than combat shotguns especially with the damage nerf. I really think it's too soon to tell if combat shotguns need a cost of 80, that seems like too much before the other changes are felt.

scrubmcnoob commented:
For the conflicting file icons/obj/storage.dmi I downloaded a new template off the git, like a hour ago, and added the box sprites to it. So I don't why it's still displaying unless it needs time to check itself.

@Fox-McCloud fixed the pathing, removed dragonbreath order from cargo, combat shotgun is 80 and new sprite for speedloaders.

Fox-McCloud commented on code/datums/supplypacks.dm:
```diff
@@ -326,20 +326,54 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
+					/obj/item/weapon/storage/belt/bandolier,
+					/obj/item/weapon/storage/belt/bandolier,
+					/obj/item/weapon/storage/belt/bandolier)
+	cost = 45
```
Increase to 50.

Fox-McCloud commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -152,6 +156,120 @@
+		new /obj/item/ammo_casing/shotgun/beanbag(src)
+		new /obj/item/ammo_casing/shotgun/beanbag(src)
+
+/obj/item/weapon/storage/box/rubbershot
```
```
/obj/item/weapon/storage/box/rubbershot
	name = "box of rubber shots"
	desc = "A box full of rubber shots, designed for riot shotguns."
	icon_state = "rubbershot_box"

/obj/item/weapon/storage/box/rubbershot/New()
	..()
	for(var/i in 1 to 7)
		new /obj/item/ammo_casing/shotgun/rubbershot(src)
```

Fox-McCloud commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -152,6 +156,120 @@
 		new /obj/item/ammo_casing/shotgun(src)
 		new /obj/item/ammo_casing/shotgun(src)
 
+/obj/item/weapon/storage/box/buck
```
```
/obj/item/weapon/storage/box/lethalshot
	name = "box of lethal shotgun shots"
	desc = "A box full of lethal shots, designed for riot shotguns."
	icon_state = "lethalshot_box"

/obj/item/weapon/storage/box/lethalshot/New()
	..()
	new /obj/item/ammo_casing/shotgun/buckshot(src)
	new /obj/item/ammo_casing/shotgun/buckshot(src)
	new /obj/item/ammo_casing/shotgun/buckshot(src)
	new /obj/item/ammo_casing/shotgun/buckshot(src)
	new /obj/item/ammo_casing/shotgun/buckshot(src)
	new /obj/item/ammo_casing/shotgun/buckshot(src)
	new /obj/item/ammo_casing/shotgun/buckshot(src)
```

Fox-McCloud commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -152,6 +156,120 @@
+		new /obj/item/ammo_casing/shotgun/stunslug(src)
+		new /obj/item/ammo_casing/shotgun/stunslug(src)
+
+/obj/item/weapon/storage/box/beanbag
```
```
/obj/item/weapon/storage/box/beanbag
	name = "box of beanbags"
	desc = "A box full of beanbag shells."
	icon_state = "rubbershot_box"

/obj/item/weapon/storage/box/beanbag/New()
	..()
	new /obj/item/ammo_casing/shotgun/beanbag(src)
	new /obj/item/ammo_casing/shotgun/beanbag(src)
	new /obj/item/ammo_casing/shotgun/beanbag(src)
	new /obj/item/ammo_casing/shotgun/beanbag(src)
	new /obj/item/ammo_casing/shotgun/beanbag(src)
	new /obj/item/ammo_casing/shotgun/beanbag(src)
```

Fox-McCloud commented:
I have 3 examples of how to implement the boxes. Their sprites can be found here: https://github.com/tgstation/tgstation/raw/master/icons/obj/storage.dmi

You may have to recolor some of the others to get it to work for the ones not included here.

The armory should have the regular boxes of ammo.

Each of the shotgun crates, however, should come with an empty shotgun speedloader.

scrubmcnoob commented:
Also fixed #7048
